src/caretogether-pwa/src/App.tsx

```diff
@@ -179,6 +184,10 @@ function App() {
   const handleDrawerClose = () => {
     setOpen(false);
   };
+  const [value, setValue] = React.useState(0);
```

Comment:
I don't think you're actually using this (see my note below), you can remove this.

Reply:
The value on the BottomNavigation component uses these to set the classes to highlight the correct button. I did however find a bug. If you from full to small size and the BottomNavigation pops up, the highlighted state isn't sync'd correctly. I'll see if I can fix that quick